### PR TITLE
chapter19_part4: /210_Identifying_Words/30_ICU_plugin.asciidoc

### PR DESCRIPTION
**Pull request by lxy4java**
Thanks for the Pull Request!  We really appreciate your help and contribution!

Before you submit the PR, have you signed Contributor License Agreement: https://www.elastic.co/contributor-agreement/ ?

PR's (no matter how small) cannot be merged until the CLA has been signed.  
It only needs to be signed once, however. Thanks!

### CODE REVIEW

File: 210_Identifying_words/30_ICU_plugin.asciidoc [hunk 1]
```diff
@@ -1,40 +1,33 @@
 [[icu-plugin]]
-=== Installing the ICU Plug-in
+=== 装载 ICU 插件
```
Comment (medcl) [inline 210_Identifying_words/30_ICU_plugin.asciidoc]: 装载 
换成安装吧

File: 210_Identifying_words/30_ICU_plugin.asciidoc [hunk 2]
```diff
@@ -1,40 +1,33 @@
...
 
-The https://github.com/elasticsearch/elasticsearch-analysis-icu[ICU analysis
-plug-in]  for Elasticsearch uses the _International Components for Unicode_
-(ICU) libraries  (see http://site.icu-project.org[site.project.org]) to
-provide a rich set of tools for dealing with Unicode.((("International Components for Unicode libraries", see="ICU plugin, installing")))((("words", "identifying", "installing ICU plugin")))((("ICU plugin, installing"))) These include the
-`icu_tokenizer`, which is particularly useful for Asian languages,((("Asian languages", "icu_tokenizer for"))) and a number
-of token filters that are essential for correct matching and sorting in all
-languages other than English.
+
+Elasticsearch的 https://github.com/elasticsearch/elasticsearch-analysis-icu[ICU 分析器插件] 使用 _国际化组件Unicode_ (ICU)函数库（详情查看 http://site.icu-project.org[site.project.org] ）提供丰富的处理Unicode工具。
```
Comment (medcl) [inline 210_Identifying_words/30_ICU_plugin.asciidoc]: 中文和英文之间需要加上空格

**Comment by lxy4java** [comment]
@medcl  已修改 谢谢 M大

**Comment by medcl** [comment]
LGTM

File: 210_Identifying_words/30_ICU_plugin.asciidoc [hunk 3]
```diff
@@ -1,40 +1,33 @@
...
-The https://github.com/elasticsearch/elasticsearch-analysis-icu[ICU analysis
-plug-in]  for Elasticsearch uses the _International Components for Unicode_
-(ICU) libraries  (see http://site.icu-project.org[site.project.org]) to
-provide a rich set of tools for dealing with Unicode.((("International Components for Unicode libraries", see="ICU plugin, installing")))((("words", "identifying", "installing ICU plugin")))((("ICU plugin, installing"))) These include the
-`icu_tokenizer`, which is particularly useful for Asian languages,((("Asian languages", "icu_tokenizer for"))) and a number
-of token filters that are essential for correct matching and sorting in all
-languages other than English.
+
+Elasticsearch的 https://github.com/elasticsearch/elasticsearch-analysis-icu[ICU 分析器插件] 使用 _国际化组件 Unicode_ (ICU) 函数库（详情查看 http://site.icu-project.org[site.project.org] ）提供丰富的处理 Unicode 工具。
+这些包含对处理亚洲语言特别有用的 `icu_分词器` ，还有很多除英语之外的必要分词过滤器，这些分词过滤器用于正确匹配和所有语言排序。
```
Comment (gitqh) [inline 210_Identifying_words/30_ICU_plugin.asciidoc]: "用于正确匹配和所有语言排序" 改为“用于在所有语言中正确匹配与排序”会否更适合

File: 210_Identifying_words/30_ICU_plugin.asciidoc [hunk 4]
```diff
@@ -1,40 +1,33 @@
...
 
 [NOTE]
 ==================================================
 
-The ICU plug-in is an essential tool for dealing with languages other than
-English, and it is highly recommended that you install and use it.
-Unfortunately, because it is based on the external ICU libraries, different
-versions of the ICU plug-in may not be compatible with previous versions.  When
-upgrading, you may need to reindex your data.
+ICU插件是处理英语之外语言的必需工具，非常推荐你安装并使用它，不幸的是，因为是基于额外的ICU函数库，
```
Comment (gitqh) [inline 210_Identifying_words/30_ICU_plugin.asciidoc]: 中英文之间建议加空格，下文中ICU建议修改格式

**Comment by lxy4java** [comment]
@gitqh  已修改，谢谢。。。

File: 210_Identifying_words/30_ICU_plugin.asciidoc [hunk 5]
```diff
@@ -1,40 +1,33 @@
...
-The https://github.com/elasticsearch/elasticsearch-analysis-icu[ICU analysis
-plug-in]  for Elasticsearch uses the _International Components for Unicode_
-(ICU) libraries  (see http://site.icu-project.org[site.project.org]) to
-provide a rich set of tools for dealing with Unicode.((("International Components for Unicode libraries", see="ICU plugin, installing")))((("words", "identifying", "installing ICU plugin")))((("ICU plugin, installing"))) These include the
-`icu_tokenizer`, which is particularly useful for Asian languages,((("Asian languages", "icu_tokenizer for"))) and a number
-of token filters that are essential for correct matching and sorting in all
-languages other than English.
+
+Elasticsearch的 https://github.com/elasticsearch/elasticsearch-analysis-icu[ICU 分析器插件] 使用 _国际化组件 Unicode_ (ICU) 函数库（详情查看 http://site.icu-project.org[site.project.org] ）提供丰富的处理 Unicode 工具。
+这些包含对处理亚洲语言特别有用的 `icu_分词器` ，还有很多除英语之外的必要分词过滤器，这些分词过滤器用于在所有语言中正确匹配与排序。
```
Comment (AlixMu) [inline 210_Identifying_words/30_ICU_plugin.asciidoc]: These include the `icu_tokenizer`, which is particularly useful for Asian languages,((("Asian languages", "icu_tokenizer for"))) and a number of token filters that are essential for correct matching and sorting in all languages other than English.
这里可不可以这样翻译（that是定语从句）：
这些包含对处理亚洲语言特别有用的 `icu_分词器` ，还有大量对除英语外其他语言进行正确匹配和排序所必须的分词过滤器。

**Comment by gitqh** [comment]
LGTM

**Comment by luotitan** [comment]
重提 pr #423

**Pull request closed without merging**